docker-compose/integrator-analytics/README.md

```diff
@@ -1,36 +1,36 @@
-# WSO2 Enterprise Integrator for Integration use-cases with Analytics support
+# WSO2 Enterprise Integrator <br> For Integration Use-cases with Analytics Support
 
-![alt tag](integrator-analytics.png)
+![alt tag](deployment-diagram.png)
 
 ## Prerequisites
 
  * [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose) are required to run this deployment.
 
 ## How to Run
 
-  1. Pull necessary images or build them using its [Dockerfile](../../dockerfiles/README.md):
+  1. Pull necessary images or build them using its [Dockerfile](../../dockerfiles):
      ```
-     docker pull wso2ei-analytics:6.1.1
      docker pull wso2ei-integrator:6.1.1
+     docker pull wso2ei-analytics:6.1.1
      ```
 
   2. Pull MySQL Docker image:
      ```
      docker pull mysql:5.7.19
      ```
 
-  3. Download the latest Docker resources release zip for the product from [releases](https://github.com/wso2/docker-ei/releases) <br>
-     page or clone this repository and switch to latest tag.
+  3. Download the latest Docker resources for the product from [releases](https://github.com/wso2/docker-ei/releases) <br>
+     page or clone this repository to your home machine and switch to latest tag.
```

Comment:
Isn't it better to call this local machine?

Reply:
I tend to agree with Chiranga. May be we can improve this in the next pull request.